### ISSUE TITLE
Commands failing with --env aragon:rinkeby

### ISSUE THREAD

**Issue by ajsantander** [aragon/aragon-cli#1090]
@dapplion @0xGabi 

I identified what causes this problem, but don't have enough context to implement a proper fix.

If I change [line 96 in configEnvironment.js](https://github.com/aragon/aragon-cli/blob/develop/packages/cli/src/lib/environment/configEnvironment.js#L96) to:
```
    apm: {
      ...merge(apm, env.apm),
      'ens-registry': env.registry,
    },

```

 I can succesfully run a command like `aragon apm packages --env aragon:rinkeby`, but tests break all around. So, I'm guessing that we need to fix that, but properly :)

We should also identify the pattern of the error and try to look for similar errors performed during logic extraction.

**Comment by 0xGabi** [comment]
Perfect, thanks @ajsantander for find that out

**Comment by 0xGabi** [comment]
Solved with #1100
